lib/index.js

```diff
@@ -23,7 +23,8 @@ module.exports = function koaXmlBody(options) {
                 options.encoding = ctx.request.charset
             }
             return parse(ctx.req, options).then(data => {
-                ctx.request[bodyKey] = data
+                ctx.request[bodyKey] = data.xml;
+                ctx.request.rawBody = data.rawBody;
```

Comment:
I'm not sure this is necessary. I will check `koa-bodyparser` soon.

Reply:
This is absolutely necessary if you need to get the xml as-is before parsing to JSON. And it will not hurt anything if you don't. I could also introduce an option to configure the behaviour and update the docs if you intend to merge this PR.